client/packages/requisitions/src/ResponseRequisition/ListView/CreateRequisitionModal.tsx

```diff
@@ -36,51 +36,50 @@ export const CreateRequisitionModalComponent: FC<
   CreateRequisitionModalProps
 > = ({ isOpen, onClose, onCreate }) => {
   const t = useTranslation();
-
+  const { Modal } = useDialog({ isOpen, onClose, disableBackdrop: false });
   const { data, isLoading } = useName.document.customers();
 
   const [customer, setCustomer] = React.useState<NameRowFragment | null>(null);
 
-  const handleCustomerChange = (customer: NameRowFragment | null) => {
-    setCustomer(customer);
-  };
+  const programTab = React.useMemo(
```

Comment:
Memorised to stop the tabs from flickering since api for names getting refetched ;x